pkg/reconciler/common/transformers.go

```diff
@@ -310,7 +310,7 @@ func SplitsByEqual(arg string) ([]string, bool) {
 // TaskImages replaces step and params images.
 func TaskImages(images map[string]string) mf.Transformer {
 	return func(u *unstructured.Unstructured) error {
-		if u.GetKind() != "ClusterTask" {
+		if u.GetKind() != "ClusterTask" && u.GetKind() != "Task" {
```

Comment:
@Aneesh-M-Bhat Are we not supporting `Task` Kind too? Seems it was supported?
cc @piyush-garg

Reply:
@jkandasa we didn't prior to this PR I think.

Reply:
yep it isn't supported prior to this.